packager/packager.js

```diff
@@ -242,6 +243,7 @@ function runServer(
   readyCallback
 ) {
   var app = connect()
+    .use(ServeStatic(options.assetRoots + '/assets', {'fallthrough':true}))
```

Comment:
Don't think fallthrough option is needed because it defaults to true.

https://github.com/expressjs/serve-static#fallthrough